config/test.exs

```diff
@@ -37,7 +37,10 @@ config :ueberauth_oidcc,
 config :screenplay, Screenplay.Repo,
   adapter: Ecto.Adapters.Postgres,
   database: "screenplay_test",
-  pool: Ecto.Adapters.SQL.Sandbox
+  pool: Ecto.Adapters.SQL.Sandbox,
+  username: "postgres",
+  password: "postgres",
```

Comment:
If you install Postgres from Homebrew, the username is your own system username and the password is blank, so I don't think this is generally applicable (indeed when I run `mix test` on this branch I get a connection failure). What prompted these changes?

Reply:
Per Slack discussion, changes in this commit have been reverted. I will open a new PR this week proposing changes that work for all users regardless of how psql was installed.